### PR TITLE
chore: add discord link

### PR DESCRIPTION
**Pull request by jayk-gupta**
added discord link of Askdevs:
https://discord.com/invite/VgFMfMJn

### CODE REVIEW

**Comment by codewithvoid** [comment]
Thanks @jayk-gupta for the PR.

I was thinking of adding link with image in a separate section. Example and code below. Can you please update. Place it after demo section.

## 👋 Join Us On Discord

<a href="https://discord.gg/DFwsFKfgEB">DiscordLogo</a>

## Code
```markdown
## 👋 Join Us On Discord

<a href="[discord.gg/DFwsFKfgEB](https://discord.gg/DFwsFKfgEB)"><img src="https://seeklogo.com/images/D/discord-logo-134E148657-seeklogo.com.png" alt="DiscordLogo" width="100" height="100"></a>
```

File: data/users/jayk-gupta.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Jay Kumar Gupta",
+  "bio": " Web Dev ",
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:3]: remove extra spaces
```json
"bio": "Web Dev"
```

File: data/users/jayk-gupta.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  "image": "https://avatars.githubusercontent.com/u/100681165?v=4",
+  "categories": [
+    {
+      "slug": "web-development",
+      "topics": ["CSS","javascript"]
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:8]: Add javascript in programming category. Example.
```json
{
  "slug": "programming",
  "topics": ["JavaScript"]
}
```

File: README.md [hunk 3]
```diff
@@ -6,6 +6,8 @@ Tweet your technical questions to tech twitter volunteers.
```
Comment (codewithvoid) [inline README.md:6]: revert this file in this PR

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by jayk-gupta** [comment]
> Thanks @jayk-gupta for the PR.
> 
> I was thinking of adding link with image in a separate section. Example and code below. Can you please update. Place it after demo section.
> 
> ## 👋 Join Us On Discord
> [DiscordLogo](https://discord.gg/DFwsFKfgEB)
> 
> ## Code
> ```gfm
> ## 👋 Join Us On Discord
> 
> <a href="[discord.gg/DFwsFKfgEB](https://discord.gg/DFwsFKfgEB)">DiscordLogo</a>
> ```

**Comment by jayk-gupta** [comment]
Updated the discord task.
I don't know whether the json file was edited or not, I have updated that too.
Thanks for the help!

File: README.md [hunk 4]
```diff
@@ -6,8 +6,14 @@ Tweet your technical questions to tech twitter volunteers.
 
 Check out the website: https://ask-devs.vercel.app/
 
+Join our discord server: https://discord.com/invite/VgFMfMJn
```
Comment (codewithvoid) [inline README.md:9]: remove this line

File: data/users/jayk-gupta.json [hunk 5]
```diff
@@ -0,0 +1,15 @@
+{
+  "name": "Jay Kumar Gupta",
+  "bio": " Web Dev",
+  "image": "https://avatars.githubusercontent.com/u/100681165?v=4",
+  "categories": [
+    {
+      "slug": "programming",
+      "topics": [""javascript"]
```
Comment (codewithvoid) [inline data/users/jayk-gupta.json:8]: can you make this change in separate branch? It should not be submitted with the discord link change.

**Review by codewithvoid — CHANGES_REQUESTED** [review]

**Comment by codewithvoid** [comment]
@jayk-gupta are you working on the comments? We need discord link change soon.

**Comment by codewithvoid** [comment]
> > @jayk-gupta are you working on the comments? We need discord link change soon.
> 
> Comments on PRs? What do I need to do exactly? I can work on this.

Yes. The PR comments. I would suggest revert data/users file and send it separately.

**Comment by jayk-gupta** [comment]
> > > @jayk-gupta are you working on the comments? We need discord link change soon.
> > 
> > 
> > Comments on PRs? What do I need to do exactly? I can work on this.
> 
> Yes. The PR comments. I would suggest revert data/users file and send it separately.

What to send seperately? My PR of adding my info,right?

**Comment by codewithvoid** [comment]
> What to send seperately? My PR of adding my info,right?
Yes. Just add discord link in this PR. (README file change)

**Comment by jayk-gupta** [comment]
> > What to send seperately? My PR of adding my info,right?
> > Yes. Just add discord link in this PR. (README file change)

okay

**Pull request closed without merging**